### ISSUE TITLE
nano ui missing

### ISSUE THREAD

**Issue by davethegr8** [zeit/hyper#1715]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: OSX 10.11.6
- **Hyper.app version**: 1.3.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/davethegr8/ee03422b43dab9c8529b425d2b796c34
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue

When using nano, the "ui" is missing (short hints). See screenshot.

![screen shot 2017-03-28 at 1 04 47 pm](https://cloud.githubusercontent.com/assets/55392/24425033/79d337a4-13b7-11e7-8ce9-9a2e61560eeb.png)

**Comment by mnpenner** [comment]
Reproduced on Ubuntu 16.04.

![image](https://cloud.githubusercontent.com/assets/1112998/24774273/297ba1de-1acd-11e7-9238-584bb15dcfe1.png)

**Comment by mdeboer** [comment]
Same here, also indenting with tabs is broken. The cursor moves but entered text isn't indented.

**Comment by chabou** [comment]
@mdeboer 
I can't reproduce and this is a old issue.
Can you give us more information (OS, Hyper version, config etc..) and how to reproduce it ?

**Comment by mdeboer** [comment]
OSX 10.12.6 (Sierra)
Hyper version: 1.4.3 (stable)
Nano:
```
GNU nano version 2.0.6 (compiled 14:26:45, Oct 14 2016)
 Email: nano@nano-editor.org    Web: http://www.nano-editor.org/
 Compiled options: --disable-nls --enable-color --enable-extra --enable-multibuffer --enable-nanorc --enable-utf8
```

I think it's more to do with ncurses as more application have this issue where black text in white bars (like when saving or searching with nano) don't appear (or appear as white for that matter).

**Comment by chabou** [comment]
Do you have a hyper theme? can you try without any plugin?

**Comment by mdeboer** [comment]
@chabou will do tomorrow 👍

**Comment by mnpenner** [comment]
![image](https://user-images.githubusercontent.com/1112998/30302145-f6f18508-9713-11e7-9c64-d2ab5ca57257.png)

Works in the latest deb build for me (1.4.3)

**Comment by mdeboer** [comment]
I haven't come round to test it without theme yet, give me an hour or so. My theme uses the vibrancy option (glass background) of Electron. Maybe that's buggy, don't know 😉

**Comment by mdeboer** [comment]
@mnpenner Okay, I tried it without any theme and then it works fine, just like your screenshot. However, when I set my background to RGBA with a low alpha channel, say 0.3 ... normal text is fine except for text in the white bars, they have the same transparency as the background ... almost like they were "cut out" of the white bars.

This would also relate to my problem with the theme I'm using ([hyperterm-dark-fusion-vibrancy](https://github.com/mdeboer/hyperterm-dark-fusion-vibrancy)) as that is using the vibrancy option of OSX to create a glass-like background. It would make sense in relation to the problem I described that whatever goes wrong and assigns the background color/transparency to the text instead of the foreground colour gets confused when vibrancy is used. On a lower level this background would be replaced entirely and thus no color can be picked from it, leaving it entirely transparent.

TL;DR;
- Text with background gets same colour as background with transparency.
- Text with background gets becomes fully transparent with vibrancy background.

You can read all about vibrancy here: https://electron.atom.io/docs/api/browser-window/

**Comment by kasper3** [comment]
On Windows, nano "experience" with hyper is exalting, `nano[Enter]` crashes hyper I am using bin from https://files.lhmouse.com/nano-win/, source https://github.com/lhmouse/nano-win

**Comment by mojoaxel** [comment]
:-1: I cannot reproduce this with 2.0.0-canary.18 on linux.

**Comment by mdeboer** [comment]
@mojoaxel go figure, this is an issue on OSX and afaik not on Linux ;)

> OS version and name: OSX 10.11.6

**Comment by davethegr8** [comment]
@mojoaxel confirmed, it also doesn't repro on 2.0.0 stable on OSX 10.13.4. This is probably safe to close for now and another issue reopened should new/more relevant causes come up. Though, I'll leave it for project members to make that determination.

**Comment by albinekb** [comment]
Can't reproduce this in 2.0.0, please re open if you can provide steps to reproduce!

**Comment by xamgore** [comment]
I use the following plugins and **zsh terminal**. Same problem, @albinekb 

![image](https://user-images.githubusercontent.com/4586392/39656494-f25be25a-5008-11e8-86df-7f9c799f2efb.png)


```js
  plugins: [
    'hyperterm-overlay',
    // 'hyperterm-alternatescroll',
    'hyperterm-paste',
    'hyperterm-dibdabs',
    'hyperlinks',
    'hyper-autohide-tabs',
    'hyperminimal',
    'hyper-disable-new-version-notification',
    'hyper-atom-dark-transparent'
  ],
```

Can share all required configuration files if you need.
